pages/blog/20200110-androidstudiosetup.md

```diff
@@ -0,0 +1,100 @@
+# How To Setup Android Studio 
+
+10, January 2020 • [LordJashin32](https://github.com/LordJashin32)
+
+---
+
+This blog is about how to setup [Android Studio](https://en.wikipedia.org/wiki/Android_Studio) for the [treehouses remote](https://github.com/treehouses/remote) repository.
+These projects rely on the [Java](https://en.wikipedia.org/wiki/Java_%28programming_language%29) programming language and Android Studio is an Integrated Development Environment (IDE) for Android Apps. 
+
+---
+
+## Download Java
+
+[JRE](https://en.wikipedia.org/wiki/Java_virtual_machine#Java_Runtime_Environment) is required for Java programming in Android Studio. 
```

Comment:
If you're developing in Java, you would need the JDK (which includes the JRE), would you not?

Anyways, I believe Android Studio comes bundled with OpenJDK, meaning you shouldn't have to download it in the first place.

Reply:
> 
> 
> If you're developing in Java, you would need the JDK (which includes the JRE), would you not?
> 
> Anyways, I believe Android Studio comes bundled with OpenJDK, meaning you shouldn't have to download it in the first place.

Yeah, you're totally right. I did a loop de loop. Okay, I remove it.